src/ProjectTemplates/Web.ItemTemplates/content/RazorComponent/.template.config/template.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "ASP.NET"
+  ],
+  "name": "Razor Component",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A Razor Component",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorComponent",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorComponent",
+  "shortname": "razorcomponent",
+  "sourceName": "Index",
```

Comment:
Index.razor isn't really appropriate here because the component item template doesn't have a page directive. Let's go with Component1.razor instead.